app/Libraries/BBCodeFromDB.php

```diff
@@ -382,21 +382,26 @@ public static function removeBlockQuotes($text)
     {
         $level = 0;
         $marker = 0;
+        $quoteBeginning = 0;
 
-        while ($marker >= 0 && $marker < mb_strlen($text) && $level >= 0) {
+        while ($marker >= 0 && $level >= 0) {
```

Comment:
This loop should be implemented as forward-only to avoid infinite loops, which are really undesirable for this method as it's being used by an unattended job.